deployment/rbac.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: node-problem-detector
```

Comment:
NPD has a built-in role in Kubernetes `system:node-problem-detector`. We can probably use that.

Reply:
done

Reply:
@Random-Liu PTAL